.github/workflows/release.yml

```
@@ -0,0 +1,34 @@
+name: Release
+on:
+    push:
+        branches:
+            - master
+jobs:
+    release:
+        name: Build and Release
+        runs-on: ubuntu-latest
+        permissions:
+            contents: write
+            issues: write
+            pull-requests: write
+            id-token: write
+        steps:
+            - name: Checkout
+              uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
+              with:
+                  fetch-depth: 0
+            - name: Setup Node.js
+              uses: actions/setup-node@b39b52d1213e96004bfcb1c61a8a6fa8ab84f3e8
+              with:
+                  node-version: 20.x
+            - name: Install Dependencies
+              run: npm clean-install
+            - name: >-
+                  Verify The Integrity of Provenance Attestations and Registry
+                  Signatures for Installed Dependencies
+              run: npm audit signatures
+            - name: Release
```

```suggestion
            - name: Release
               if: success()
```
don't we need this condition checking? 🤔

---

it has this check by default

---

I don't think we need it @shayan-deriv, as far as I remember, when  any of the of the steps fails, the rest are skipped